docs/docs/server-administration/dns.md

```diff
@@ -4,42 +4,56 @@
 With the release of version 1.7.0, we have implemented support for DNSSEC. DNSSEC requires a Master -> Slave setup. IF the existing implementation is a Master <-> Master setup, it is not supported. DNSSEC also requires at least Ubuntu 22.04 or Debian 11!
 :::
 
-## Host your DNS on Hestia
+## Host DNS for your domain on Hestia
 
-[Create a DNS Zone](../user-guide/dns.md#adding-a-dns-zone) with the **child-ns** template, then login to your domain registrar’s panel and change the name servers of the domain. Depending your registrar panel, you could be able to create glue records. You may need to wait for up to 24 hours before the name servers become active.
+Pre-requisites
 
-## DNS Cluster setup
+These steps require that you configure the DNS servers of your domain to use your Hestia servers.
+
+- Note that most domain providers require two or more DNS servers to be configured.
+- The name servers will most likely be required to be registered as 'Glue records'
+- You may need to wait for up to 24 hours before the name servers become available
+
+Preparing the domain and DNS
+
+1. On your Hestia master, [create a DNS Zone](../user-guide/dns.md#adding-a-dns-zone) with the **child-ns** template
+2. On your domain registrar panel, set the name servers of the domain to the Hestia servers
+
+If you are looking at options to minimise DNS-related downtime or for a way to automatically synchronise DNS zones across all your servers, you might consider setting up a DNS cluster.
+
+If DNSSEC matters to you, then you must use Master -> Slave. However if you would like to add zones to either server and have them replicate to the other, then configure as Master <-> Master.
 
 ::: tip
-Create for each server a unique user and assing them the "Sync DNS User" or "dns-cluster" role!
+If you have just set up your slave, check that the host name resolves and that you have a valid SSL certificate
 :::
 
-If you are looking at options to minimise DNS-related downtime or for a way to manage DNS across all your servers, you might consider setting up a DNS cluster.
 
-1. Whitelist your master server IP in **Configure Server** -> **Security** -> **Allowed IP addresses for API**, otherwise you will get an error when adding the slave server to the cluster.
-2. Enable API access for admins (or all users).
-3. Create an API key under the **admin** user with at least the **sync-dns-cluster** permission.
+## DNS Cluster setup
+
+A Master server is where DNS zones are created, and a Slave server recieves the zone via the API. Hestia can be configured as Master <-> Master and Master -> Slave. With a Master <-> Master configuration, each Master is also a Slave, so it could be considered as Master/Slave <-> Master/Slave.
```

Comment:
```suggestion
A Master server is where DNS zones are created, and a Slave server recieves the zone via the API. Hestia can be configured as Master <-> Master or Master -> Slave. With a Master <-> Master configuration, each Master is also a Slave, so it could be considered as Master/Slave <-> Master/Slave.
```